omnibus/config/chefdk_overrides.rb

```diff
@@ -0,0 +1,23 @@
+override :berkshelf, :version => "v4.3.1"
```

Comment:
yeah. Totally get it is meant to translate to an override in the project file. Its just that if you didn't know any better, one would wonder why do I see some overrides in the project file and other in the overrides file? You and I know that the overrides for gems go in the overrides file, but if the overrides file used a `gem_override` key, I'm thinking that would be more obvious.